src/Library/demos/Web View/main.blp

```diff
@@ -53,7 +54,8 @@ Adw.Clamp {
       }
     }
 
-    Adw.Bin container {
+    WebKit.WebView web_view {
```

Comment:
@AkshayWarrier FYI this fixes the bug which prevented using `WebKit.WebView` from Blueprint :)

Reply:
Oh this is great! I'll make the same change in https://github.com/sonnyp/Workbench/pull/358 to make use of WebView from Blueprint instead of code once this is merged.